en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,122 @@
+# Publish to ByteDance Mini Games
+
+Cocos Creator starts to support publishing games to **ByteDance Mini Games** from v2.4.2.
+
+The runtime environment of ByteDance Mini Game is an extension of ByteDance Mini Program, which provides the encapsulation of WebGL interfaces on top of the Mini Program environment, making the rendering ability and performance substantially improved. However, since these interfaces are encapsulated by the ByteDance team through self-developed native implementations, they are not equivalent to the browser environment.
+
+As the engine developer, in order to simplify the workload of developers as much as possible, the main work we have done for developers includes:
+
+- The engine framework is adapted to the ByteDance API and is purely at the game logic level, so developers do not need any additional modifications
+- Cocos Creator editor provides a quick packaging process to publish directly as a ByteDance 
+- Automatically loads emote assets, cached assets and the versioning of cached assets
+
+For details about ByteDance  access, please refer to [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration).
+
+## Preparation
+
+- Download [ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download) and install it.
+
+- Refer to the document of [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration) to register, login and apply for mini games on ByteDance Developer Platform.
+
+- After the application is approved, you can find the appid in **Development Management -> Development Settings** of the developer platform.
+
+  ![](./publish-bytedancegame/appid.png)
+
+## Publishing Process
+
+Use Cocos Creator to open the project to be published, open the **Build...** panel from **Menu Bar -> Project**, select **ByteDance Mini Game** in **Platform**, fill in the appid, and click **Build**.
+
+![](./publish-bytedancegame/build.png)
+
+### Parameter Configuration
+
+The specific rules for filling in the relevant parameter configuration are as follows.
+
+- **The Compression Type of the Main Package**
+
+  To set the compression type of the main package, please refer to the document [Asset Bundle - Compression Type](../asset-manager/bundle.md#compression-type).
```

Comment:
the document [Asset Bundle - Compression Type](../asset-manager/bundle.md#compression-type).
-->
the [Asset Bundle - Compression Type](../asset-manager/bundle.md#compression-type) documentation.